demo/two-containers.job.yaml

```diff
@@ -6,10 +6,10 @@ spec:
   template:
     spec:
       containers:
-      - name: count
+      - name: container1
         image: ubuntu
         command: ["sh", "-c", "for i in `seq 1 10`; do echo $i; sleep 1; done"]
-      - name: count2
+      - name: container2
```

Comment:
could rename the container in 'single-container.yaml as well

Reply:
done